apps/abstraxion-settings/src/hooks/useAbstraxionSigningClient.ts

```diff
@@ -30,11 +30,21 @@ export const useAbstraxionSigningClient = () => {
     undefined,
   );
 
-  async function ethSigningFn(msg: any) {
-    if (!window.ethereum) {
-      alert("Please install the Metamask wallet extension");
+  async function okxSignArb(
+    chainId: string,
+    account: string,
+    signBytes: Uint8Array,
+  ) {
+    if (!window.okxwallet) {
+      alert("Please install the OKX wallet extension");
       return;
     }
+    await window.okxwallet.keplr.enable("xion-testnet-1");
```

Comment:
It's something I realized I did in "dashboard" as well, but we should step away from hardcoded value here as well as line 98. Doesn't hurt as of right now, but will need the dynamic nature of "chainId" vs "xion-testnet-1"